uploadedimage/uploadedimage.go

```diff
@@ -0,0 +1,35 @@
+package uploadedfile
+
+type UplodedFile struct {
+    filename string
+    path     string
+    mime     string
+}
+
+func (this *UplodedFile) GetFilename() string {
+    return this.filename
+}
+
+func (this *UplodedFile) GetPath() string {
+    return this.path
+}
+
+func (this *UplodedFile) GetMime() string {
+    return this.mime
+}
+
+func (this *UplodedFile) FileSize() (int, error) {
```

Comment:
shouldn't this be read in the constructor and saved in a field like the `mime` is?